src/lsp/ProjectManager.ts

```diff
@@ -284,7 +284,7 @@ export class ProjectManager {
                         projectPath: s`${projectPath}`,
                         workspaceFolder: s`${workspaceConfig.workspaceFolder}`,
                         excludePatterns: workspaceConfig.excludePatterns,
-                        enableThreading: workspaceConfig.enableThreading
+                        enableThreading: workspaceConfig.languageServer.enableThreading
```

Comment:
>enableThreading was never enabled due to incorrect workspace config preparation using unsafe casting.

Wow, nice catch! I just verified this is definitely a problem in the published vscode extension. Our huge app is running both our projects on the same "main" thread instead of in background threads. Yikes!